lib/Sema/TypeCheckPropertyWrapper.cpp

```diff
@@ -462,10 +462,17 @@ AttachedPropertyWrappersRequest::evaluate(Evaluator &evaluator,
       ctx.Diags.diagnose(attr->getLocation(), diag::property_wrapper_computed);
       continue;
     }
-
+    
     result.push_back(mutableAttr);
   }
 
+  // TODO: Property wrapper compositions do not yet compose correctly
+  // in all situtaions.
+  if (result.size() > 1) {
+    ctx.Diags.diagnose(result[result.size() - 2]->getLocation(),
+                       diag::property_wrapper_composition_not_implemented);
+  }
```

Comment:
Nitpick: If you do this after the `std::reverse` below, you can avoid the weird "second-to-last" computation above, which confused me until I read the comment below.